content/blog/release-2.2.1.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Qbs 2.2.1 released
```

Comment:
Since this blog post is actually about what's new in Qbs 2.2, I would suggest keeping with the title "Qbs 2.2 released" and in the first paragraph below, in the [original post](https://www.qt.io/blog/qbs-2.2-released) on the Qt blog.